config/brakeman.ignore

```diff
@@ -322,6 +276,29 @@
       ],
       "note": ""
     },
+    {
```

Comment:
Brakeman complains about our sql query in `app/queries/get_activity_log_events.rb` this just updates the ignore because we changed the query